hibernate5/src/main/java/com/baeldung/hibernate/HibernateMultiTenantUtil.java

```diff
@@ -97,19 +97,21 @@ private static Properties getProperties() throws IOException {
         return providerMap;
     }
 
-    private static Object tenantUrl(String originalUrl, String tenant, String tenantStrategy) throws UnsupportedTenancyException {
-        if (tenantStrategy.toUpperCase()
-            .equals("DATABASE")) {
-            return originalUrl.replace(DEFAULT_DB_NAME, tenant);
-        } else if (tenantStrategy.toUpperCase()
-            .equals("SCHEMA")) {
-            return originalUrl + String.format(SCHEMA_TOKEN, tenant);
-        } else {
-            throw new UnsupportedTenancyException("Not yet supported");
-        }
-    }
-
-    public static final String SCHEMA_TOKEN = ";INIT=CREATE SCHEMA IF NOT EXISTS %1$s\\;SET SCHEMA %1$s";
-    public static final String DEFAULT_DB_NAME = "mydb1";
+//    private static Object tenantUrl(String originalUrl, String tenant, String tenantStrategy) throws UnsupportedTenancyException {
```

Comment:
Let's remove all the commented-out code.

Reply:
Ok